virttest/utils_windows/wmic.py

```diff
@@ -0,0 +1,71 @@
+"""
+Windows WMIC utilities
+"""
+
+import re
+
+
+_WMIC_CMD = "wmic"
+
+FMT_TYPE_LIST = "/format:list"
+
+
+def make_query(cmd, cond=None, props=None, get_swch=None, gbl_swch=None):
+    """
+    Make a WMIC query command. The command pattern is:
+
+      wmic [GBL_SWCH] CMD [where COND] get [PROPS] [GET_SWCH]
+
+    :param cmd: WMIC command.
+    :param cond: Query condition to be appended to `where`.
+    :param props: Properties to be get.
+    :param get_swch: Local switch of `get`.
+    :param gbl_swch: Global switch.
+
+    :return: Query command.
+    """
+    query = [_WMIC_CMD, cmd]
+    if gbl_swch:
+        query.append(gbl_swch)
+    if cond:
+        query.append("where (%s)" % cond)
+    query.append("get")
+    if props:
+        query.append(",".join(props))
+    if get_swch:
+        query.append(get_swch)
+    return " ".join(query)
+
+
+def noinstance(data):
```

Comment:
Please use a  more specific function name

Reply:
OK, will change its name to `is_noinstance`.